server/src/tests/ws.rs

```diff
@@ -802,3 +802,39 @@ async fn notif_is_ignored() {
 	// This call should not be answered and a timeout is regarded as "not answered"
 	assert!(client.send_request_text(r#"{"jsonrpc":"2.0","method":"bar"}"#).with_default_timeout().await.is_err());
 }
+
+#[tokio::test]
+async fn drop_client_with_pending_calls_works() {
+	init_logger();
+
+	let (handle, addr) = {
+		let server = ServerBuilder::default().build("127.0.0.1:0").with_default_timeout().await.unwrap().unwrap();
+
+		let mut module = RpcModule::new(());
+
+		module
+			.register_async_method("infinite_call", |_, _| async move {
+				futures_util::future::pending::<()>().await;
```

Comment:
This a really bad and shouldn't be used but just a way to ensure that the futures won't ever complete.